### ISSUE TITLE
I miss the watchdog

### ISSUE THREAD

**Issue by krambriw** [andrewjfreyer/monitor#133]
Hello Andrew,
I miss the watchdog in your latest version. For me this is necessary since monitor is one of many  services in my home automation system and I supevise the health of all of them. I f I remember correctly it was param -m in earlier versions

Kind regards, Walter

**Comment by andrewjfreyer** [comment]
I removed this feature after [this](https://community.home-assistant.io/t/monitor-reliable-multi-user-distributed-bluetooth-occupancy-presence-detection/68505/15) poll. It seemed that among the users that responded, the quantity of options was overwhelming, so I simplified. 

I'll meet in the middle and push a new version that causes `monitor` to respond to an mqtt message for health.

**Comment by krambriw** [comment]
That is excellent, thank you!

**Comment by andrewjfreyer** [comment]
Implemented an echo function. See the FAQ.

**Comment by krambriw** [comment]
Dear Andrew, thank you, I have upgraded to this latest version. But I get errors at initial run. Have you raised the requirements? It seems this will require a more complex upgrade than just upgrading bash and mosquitto. Checking with apt-cache policy bash and apt-cache policy mosquitto shows I',m updated (currently running Jessie on a RPi and I have no plans to upgrade to Stretch)

BR Walter

> error: minimum required bash version 4.4+ not found. please update.
> error: minimum required mosquitto_sub version 1.5+ not found. please update.
> error: minimum required mosquitto_pub version 1.5+ not found. please update.

**Comment by andrewjfreyer** [comment]
I’m surprise Jesse is lower than 4.4.  I can remove the bash requirement,  but you can safely update to mosquito 1.5 via the instructions on the readme. Change stretch to Jesse where appropriate and it should work. 

Any reason you prefer the older software? I’m curious.

**Comment by andrewjfreyer** [comment]
I've removed the requirement for bash 4.4 for you. it's optional for now, but there's a possiblity that it'll be a requirement in a future update. 

Please follow the installation instructions in the readme to install mosquito  1.5, that is a requirement that I'm not going to roll back.

**Comment by krambriw** [comment]
Dear Andrew, thank you so much, your monitor solution is really working excellent! I managed to upgrade mosquitto following your instructions, it worked as it should

>Any reason you prefer the older software? I’m curious.

In general, I do updates but in this particular case I wanted to run your monitor solution in one of my 8 distributed Pi's (making up our Home Automation & Video Monitoring system) that already had some other complex software (Motion and OpenCV) installed. Earlier that took me quite a while to install, build and configure. Besides I had already tried to upgrade the RPi to Stretch but it failed at the very end, messing up my whole python environment. I could however easily recover since I use additional sd cards attached to all my Pi's and make regular sd card copies with the built in tool available in Raspbian

I could of course also easily devote another Pi for this task, that would not be a big problem but I thought I would like to use the already available one since it was located in a good position in the house for monitoring arrivals/departures

So once again, thank you for a great BT monitoring solution,
Kind regards, Walter
